M2/Macaulay2/packages/GKMManifolds/Documentation_GKMManifolds.m2

```diff
@@ -0,0 +1,2165 @@
+			(ii) $X$ has finitely many $T$-fixed points, and (iii) $X$ has finitely
+			many one-dimensional $T$-orbits.  The data of the zero and one dimensional
+			$T$-orbits of $X$ define the moment graph of $X$, with which one can carry out
+			$T$-equivariant cohomology and $T$-equivariant $K$-theory computations by
+			combinatorial means.  This package provides methods for these computations
+			in Macaulay2.
+			
+		Text
+			For mathematical background see:
+			
+			@UL{
+			{"[BM01] T. Braden and R. MacPherson.  From moment graphs to intersection cohomology.  Math. Ann. 321 (2001), 533-551."},
+			{"[BGH02] E. Bolker, V. Guillemin, and T. Holm.  How is a graph like a manifold?  arXiv:math/0206103."},
+			{"[CDMS18] A. Cameron, R. Dinu, M. Michalek, and T. Seynnaeve.  Flag matroids: algebra and geometry.  arXiv:1811.00272."},
+			{"[DES20] R. Dinu, C. Eur, and T. Seynnaeve.  K-theoretic Tutte polynomials of morphisms of matroids.  arXiv:math/2004.00112."},
+			{"[FS12] A. Fink and S. Speyer.  K-classes for matroids and equivariant localization.  Duke Math. J. 161 (2012), no. 14, 2699-2723."},
+			{"[GKM98] M. Goresky, R. Kottwitz, and R. MacPherson.  Equivariant cohomology, Koszul duality, and the localization theorem. Invent. Math. 131 (1998), no. 1, 25-83."},
+			{"[RK03] I. Rosu.  Equivariant K-theory and equivariant cohomology.  With an Appendix by I. Rosu and A. Knutson.  Math. Z. 243 (2003), 423-448."},
+			{"[Tym05] J. Tymoczko.  An introduction to equivariant cohomology and homology, following Goresky, Kottwitz, and MacPherson.  Contemp. Math. 388 (2005), 169-188."},
+			{"[VV03] G. Vezzosi and A. Vistoli.  Higher algebraic K-theory for actions of diagonalizable groups. Invent. Math. 153 (2003), no. 1, 1–44."}
+			}@
+
+		Text
+			In this package, while GKM manifolds are special kinds of T-varieties, we use the two terms 
+			"GKM manifolds" and "T-varieties" interchangeably.  For instance, @TO TVariety@ is the class 
+			of GKM manifolds, @TO TKClass@ the class of T-equivariant K-classes on a GKM manifolds, and 
+			@TO TMap@ the class of T-equivariant maps between T-varieties.
+
+		Text
+			@SUBSECTION "Contributors"@
+		Text
+			The following people have contributed code, improved existing code, or enhanced the documentation:
+			@HREF("https://www.mis.mpg.de/combag/members/tim-seynnaeve.html","Tim Seynnaeve")@.
+		
+	SeeAlso
+		"Example: generalized flag varieties"
+		"Example: smooth toric varieties"
+
+		
+
+///
+
+
+
+doc ///
+	Key
+		"Example: generalized flag varieties"
+	Description
+		Text
+			Let $G$ be a reductive complex Lie group and $P$ a 
+			parabolic subgroup containing a maximal torus $T$.  The generalized flag variety $G/P$ is a GKM manifold 
+			with the action of $T$.  This package allows users to create a generalized flag variety for classical Lie types 
+			($A$, $B$, $C$, and $D$) as a @TO "TVariety"@ with conventions explicitly laid out as follows.
+
+		Text
```

Comment:
Removing lines such as this line "Text" would probably change nothing.  Is there a reason for inserting them?  (It's not intended to separate paragraphs.)